lib/node_modules/@stdlib/stats/base/dnanminabs/manifest.json

```diff
@@ -24,6 +26,46 @@
   ],
   "confs": [
     {
+      "task": "build",
+      "src": [
+        "./src/dnanminabs.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lm"
+      ],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/abs",
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int64",
+        "@stdlib/napi/argv-strided-float64array",
+        "@stdlib/napi/create-double"
+      ]
+    },
+    {
+      "task": "benchmark",
+      "src": [
+        "./src/dnanminabs.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lm"
+      ],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/abs",
```

Comment:
Why is this dependency needed, if it wasn't previously? Same question on L41.

Reply:
i have removed it now